Makefile

```diff
@@ -460,7 +460,7 @@ PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacO
 PKGDIR=out/dist-osx
 
 release-only:
-	@if `grep -q REPLACEME doc/api/*.md`; then \
+	@if [ "$(DISTTYPE)" = "release" ] && [ `grep -q REPLACEME doc/api/*.md` ]; then \
```

Comment:
It's based on the assumption that people use it for internal releases.  We do that at IBM although I don't know if we use this particular mechanism.